app/Assets/Http/Controllers/AssetManageController.php

```diff
@@ -104,7 +104,7 @@ public function postCreate(CreateRequest $request)
             $resource,
             \Auth::user(),
             \Request::get('name'),
-            \Request::get('description')
+            \Request::get('description') == null ? "" : \Request::get('description')
```

Comment:
That only works if that field does not return anything in the request. The request has description as null so this will fail.